src/CMakeLists.txt

```diff
@@ -348,11 +374,13 @@ set(main_exe_sources
 
 
 add_executable(${main_exe_name} ${main_exe_sources})
+# target_link_libraries(${main_exe_name} PRIVATE
+#     SDL2::Main
+#     rigel_core
+#     Boost::boost
+#     Boost::program_options
+#     Boost::disable_autolinking
+# )
 target_link_libraries(${main_exe_name} PRIVATE
-    SDL2::Main
```

Comment:
I have some ideas on how to split things up in a way that would allow us to define a different main target when building with Emscripten, I'll add more details some time later!